lux/executor/PandasExecutor.py

```diff
@@ -113,28 +106,28 @@ def execute(vislist: VisList, ldf: LuxDataFrame, approx=False):
         None
         """
 
-        PandasExecutor.execute_sampling(ldf)
+        filter_executed = {}
```

Comment:
Not sure if I understand why we need to store `filter_executed` as a dictionary and what its function is.

Reply:
Yes, this was some old code when I had split this into two different for loops. I'll put it back to using just a variable to save it.